src/mrinufft/operators/base.py

```diff
@@ -753,7 +781,7 @@ def _grad_sense(self, image_data, obs_data):
 
         dataf = image_data.reshape((B, *XYZ))
         obs_dataf = obs_data.reshape((B * C, K))
-        grad = np.empty_like(dataf)
+        grad = np.zeros_like(dataf)
```

Comment:
This was such a hard catch :P We missed this as the test assert_correlate would pass if we have `nan`

Reply:
nice catch !